documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -2,24 +2,18 @@
 
 ## Version 10.0.0
 
+[company_name] [solution_name] has been renamed [detect_product_long]. As such, page links and documentation URLs have been updated accordingly. Update any [solution_name] documentation bookmarks you may have.
+
 ### New features
 
 * The npm package.json detector now performs additional parsing when attempting to find dependency versions. This can result in additional matches since versions like `^1.2.0` will now be extracted as `1.2.0` instead of as the raw `^1.2.0` string. In the case where multiple versions for a dependency are discovered, the earliest version will be used.
 * Support for Python has now been extended with Pip 24.2, Pipenv 2024.0.1, and Setuptools 74.0.0.
-
-## Version 9.10.0
+* Support for npm has been extended to 10.8.2. Node.js support has been extended to 22.7.0.
```

Comment:
Little condensing suggestion: Support for npm has been extended to 10.8.2 and Node.js 22.7.0.